.env.ci

```
@@ -37,3 +37,6 @@ NOTIFICATION_CONTEXT_SERVICE_URL=irap.example.com
 # Profile card
 PROFILE_CARD_URL=
 PROFILE_CARD_JS=
+
+# Contract for sharing sharepoint link
+DATA_PROTECTION_OSS_SHAREPOINT_URL=https://dbis.sharepoint.com/sites/DataProtectionOSS/SitePages/Homepage.aspx
```

The reason for moving this to an env var is to remove this URL from the codebase. Can we set the CI default to a fake URL, `example.com` or something?

---

yeah that is what I thought and it left me a bit puzzled, I'll do that in a sec